README.md

```diff
@@ -40,6 +44,9 @@ happen:
   currently present on the screen. When saving is done, the promise resolves.
 * After the promise resolves, we call `changeBodyBackground()`, which changes
   the `body` element's background to blue, and start the document transition.
+  The browser defers painting any updates to the DOM until
+  `documentTransition.start` is called. This allows setting up the final frame
+  for the transition asynchronously. 
```

Comment:
Maybe also mention that we resume things without a transition if start is not called

Reply:
Done.